screeps-start.js

```diff
@@ -128,11 +128,20 @@ const start = async () => {
   writeModsConfiguration();
 
   const screeps = require("@screeps/launcher");
+  const options = {
+    steam_api_key: process.env.STEAM_KEY || config.steamKey,
+    storage_disable: false,
+  };
+
+  const keys = ['runner_threads', 'processors_cnt', 'log_console'];
+  for (const key of keys) {
+    if (options[key]) {
```

Comment:
This line only overrides if the option is truthy. We should just check if it exists.
```suggestion
    if (key in options) {
```